sunkit_instruments/iris/tests/test_sji.py

```diff
@@ -1,208 +1,209 @@
-# -*- coding: utf-8 -*-
```

Comment:
What is gonna happen to this file?

Reply:
@nabobalis Dan instructed me to comment this file out as a giant docstring for now

Reply:
Sure